src/System.IO.FileSystem/tests/File/Copy_str_str_b.cs

```diff
@@ -290,31 +289,31 @@ public static void runTest()
                 printerr("Error_28vc8! Unexpected exception, exc==" + exc.ToString());
             }
 
-/* Scenario disabled while porting because it accesses a file outside the test's working directory
-#if !TEST_WINRT  // Cannot access root
-            // [] Unecessary long but valid string
+            /* Scenario disabled while porting because it accesses a file outside the test's working directory
+            #if !TEST_WINRT  // Cannot access root
+                        // [] Unecessary long but valid string
 
-            strLoc = "Loc_478yb";
+                        strLoc = "Loc_478yb";
 
-            File.Delete("\\TestFile.tmp");
+                        File.Delete("\\TestFile.tmp");
 
-            File.Create(filName).Dispose();
-            fil2 = new FileInfo(filName);
+                        File.Create(filName).Dispose();
+                        fil2 = new FileInfo(filName);
 
-            File.Copy(fil2.Name, "..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\TestFile.tmp", false);
-            fil1 = new FileInfo(fil2.FullName.Substring(0, fil2.FullName.IndexOf("\\") + 1) + fil2.Name);
-            iCountTestcases++;
-            if (!fil1.FullName.Equals(fil2.FullName.Substring(0, fil2.FullName.IndexOf("\\") + 1) + fil2.Name))
-            {
-                Console.WriteLine(fil1.FullName);
-                iCountErrors++;
-                printerr("Error_298gc! Incorrect fullname set during copy");
-            }
-            new FileInfo(filName).Delete();
-            fil1.Delete();
-            File.Delete("\\TestFile.tmp");
-#endif
-*/
+                        File.Copy(fil2.Name, "..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\TestFile.tmp", false);
+                        fil1 = new FileInfo(fil2.FullName.Substring(0, fil2.FullName.IndexOf("\\") + 1) + fil2.Name);
+                        iCountTestcases++;
+                        if (!fil1.FullName.Equals(fil2.FullName.Substring(0, fil2.FullName.IndexOf("\\") + 1) + fil2.Name))
+                        {
+                            Console.WriteLine(fil1.FullName);
+                            iCountErrors++;
+                            printerr("Error_298gc! Incorrect fullname set during copy");
+                        }
+                        new FileInfo(filName).Delete();
+                        fil1.Delete();
+                        File.Delete("\\TestFile.tmp");
+            #endif
+            */
```

Comment:
I haven't looked at the File classes yet beyond fixing the temporary files issue, but at a glance it looks like it should be removed. I think the code formatter made my changes in this section.